pygmsh/common/size_field.py

```diff
@@ -0,0 +1,67 @@
+import gmsh
+
+
+class BoundaryLayer:
+    def __init__(
+        self,
+        lcmin,
+        lcmax,
+        distmin,
+        distmax,
+        edges_list=None,
+        faces_list=None,
+        nodes_list=None,
+        NumPts_PerCurve=100,
```

Comment:
Let's rename this to `num_points_per_curve`.

Reply:
Also, have it default to `None`.

Reply:
No problem. Let me change this